ngi_pipeline/conductor/launchers.py

```diff
@@ -20,7 +20,15 @@ def get_engine_for_bp(project, config=None, config_file_path=None):
     :param NGIProject project: The project to get the engine from.
     """
     charon_session = CharonSession()
-    best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
+    try:
+        best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
```

Comment:
when this will raise? whenever there is no `project.project_id`? Or is it that `charon_session.project_get(project.project_id)["best_practice_analysis"]` can be empty?

And by the way I'm not sure that this is the most appropriate error message :sweat_smile: